enterprise/frontend/src/metabase-enterprise/embedding_iframe_sdk_setup/components/SdkIframeEmbedPreview.tsx

```diff
@@ -186,8 +186,9 @@ export const SdkIframeEmbedPreview = () => {
               : undefined,
           }),
         )
-        // TODO(EMB-869): add Metabot experience to embed flow
-        .with({ componentName: "metabase-metabot" }, () => null)
+        .with({ componentName: "metabase-metabot" }, (s) =>
```

Comment:
can we not use abbreviated names 😃 ? I have to open the whole file to see what `s` actually is.

Reply:
alright that's understandable.